svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -85,50 +68,26 @@ class LoadOutputAsLayerDialog(QDialog, FORM_CLASS):
     def __init__(self, iface, output_type=None, path=None, mode=None):
 
         # sanity check
-        if (output_type is not None
-                and output_type not in OQ_ALL_LOADABLE_TYPES):
+        if output_type not in OQ_ALL_LOADABLE_TYPES:
             raise NotImplementedError(output_type)
         self.iface = iface
         self.path = path
-        self.npz_file = None
+        # self.npz_file = None
         self.output_type = output_type
```

Comment:
I prefer to avoid having the attribute `npz_file` in all subclasses, because some of them do not load data from npz, but from csv. So I am deleting it.